pyproject.toml

```diff
@@ -16,6 +16,7 @@ dependencies = [
     "xarray",
     "markdown",
     "jinja2",
+    "scipy",
```

Comment:
I am not sure if the should add scipy as a general dependency or rather at it to [test] and the put the import statement inside the function body of _partial_duration_series()

Reply:
MIKE IO has scipy as a dependency, so unless we remove scipy from mikeio, or remove mikeio from modelskill, it is already a dependency.

Reply:
Comparing to Xarray which doesn't have scipy as a dependency, but it is still necessary for some operations
https://docs.xarray.dev/en/stable/user-guide/interpolation.html#interpolating-data
😐

Reply:
If mikeio has scipy as a dependency then modelskill can as well have the same. @daniel-caichac-DHI just keep scipy here.